features/eolearn/features/clustering.py

```diff
@@ -1,5 +1,12 @@
 """
 Module for computing clusters in EOPatch
+
+Credits:
+Copyright (c) 2020 Beno Šircelj (Josef Stefan Institute)
```

Comment:
usually you have them sorted by end dates

Reply:
I wouldn't change this for now as the first one was the main contributor.